extensions/adobe/experience/content-intelligence/webelement.schema.json

```diff
@@ -0,0 +1,125 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/content-intelligence/webelement",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Intelligence Web Element",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Web element metadata",
+  "definitions": {
+    "webelement": {
+      "properties": {
+        "xdm:id": {
+          "title": "Element Id",
+          "type": "string",
+          "meta:immutable": true,
+          "meta:usereditable": false,
+          "description": "The unique ID of the web element."
+        },
+        "xdm:name": {
+          "meta:usereditable": false,
+          "title": "Asset name",
+          "type": "string"
+        },
+        "xdm:path": {
+          "meta:usereditable": false,
+          "title": "Asset path",
+          "type": "string"
+        },
+        "xdm:modifyDate": {
```

Comment:
you should reuse `repo:lastModifiedDate` here.

Reply:
or `dc:modifyDate`